aws-organizations-organizationalunit/src/test/java/software/amazon/organizations/organizationalunit/CreateHandlerTest.java

```diff
@@ -83,7 +85,9 @@ public void handleRequest_SimpleSuccess() {
         assertThat(response.getResourceModel().getName()).isEqualTo(TEST_OU_NAME);
         assertThat(response.getResourceModel().getArn()).isEqualTo(TEST_OU_ARN);
         assertThat(response.getResourceModel().getId()).isEqualTo(TEST_OU_ID);
-        assertThat(TagTestResourcesHelper.tagsEqual(response.getResourceModel().getTags(), TagTestResourcesHelper.defaultTags));
+        assertThat(TagTestResourcesHelper.tagsEqual(
+                TagsHelper.convertOrganizationalUnitTagToOrganizationTag(response.getResourceModel().getTags()),
+                TagTestResourcesHelper.defaultTags)).isTrue();
```

Comment:
Without the `isTrue()`, this assert was never running. After adding it, this started failing. On debugging, realized the difference that both the set parameters have different types of Objects - Org Tag and OU Tag. Fixed that and now this assert is passing.